src/components/social-element/social-element.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers/utils";
+
+import * as styles from "./social-element.module.scss";
+
+const SocialElement = ({ className, links }) => {
+
+  return (
+    <div className={classnames(styles.container, className)}>
+      {links.map((item) => 
+        <a 
+          href={item.path} 
+          target="_blank" 
+          rel="noopener noreferrer" 
+          className={classnames(styles.socialIcon, {[styles.blogIcons]: item.id === 'blog'})}>
+            {item.icon}
+        </a>
+      )}
+    </div>
+  );
+};
+
+export { SocialElement };
+
+SocialElement.propTypes = {
+  className: PropTypes.string, 
+  links: PropTypes.array.isRequired,
```

Comment:
Please, add more information about the element of the array: object with key path (with value string) and icon (with value icon)

Reply:
The type icon doesn't exist, but, when I display it in console, I saw object typeof Symbol (react.Element)

Reply:
Done?